### ISSUE TITLE
Results on SDD dataset

### ISSUE THREAD

**Issue by LinneLee** [cocoon2wong/SocialCircle#14]
I tried several training strategies of the social circle model on SDD dataset, and got the best results of (ADE 6.72, FDE 10.48) with the following args:
{"#---------- AgentArgs ----------":0,
"K":1,
"K_train":1,
"Kc":20,
"T":"fft",
"anntype":"coordinate",
"batch_size":512,
"dataset":"SDD",
"depth":16,
"deterministic":0,
"epochs":500,
"feature_dim":128,
"interval":0.4,
"key_points":"0_6_11",
"log_dir":"./logs/20250915-153533modelevscsdd",
"loss":"keyl2",
"lr":1e-05,
"max_agents":50,
"model":"evsc",
"model_name":"model",
"model_type":"agent-based",
"obs_frames":8,
"only_process_trajectory":0,
"pmove":-1,
"pred_frames":12,
"preprocess":"100",
"save_base_dir":"./logs",
"split":"sdd",
"start_test_percent":0.0,
"step":1.0,
"test_step":1,
"#---------- SocialCircleArgs ----------":1,
"Ts":"none",
"partitions":8,
"rel_speed":0,
"use_direction":1,
"use_distance":1,
"use_move_direction":0,
"use_velocity":1}
The results (ADE 6.72, FDE 10.48) are close but still a little worse than the published results, and no pretrained model weights on the SDD dataset are provided. Would you mind providing the training parameters on the SDD dataset? Or the pretrained weights of the SDD dataset? Thank you very much.

**Comment by cocoon2wong** [comment]
Hi @LinneLee !

Thank you for your questions!

We have not provided our pretrained weight on SDD for the PyTorch code, since they are produced with the TensorFlow verison of code (weights can be found at https://github.com/cocoon2wong/SocialCircle/releases/tag/V1.0).

If you want to train our models on SDD with the provide PyTorch code, the setting files of our subsequent work SocialCircle+ could be used as references. Their weights are available at https://github.com/cocoon2wong/Project-Monandaeg/tree/SocialCirclePlus.

Specifically, these settings could be helpful for training on SDD (since SocialCircle and SocialCircle+ share similar structures):
(from https://github.com/cocoon2wong/Project-Monandaeg/tree/SocialCirclePlus/evspcsdd_adaptive)

```json
{"#---------- AgentArgs ----------":0,
"K":1,
"K_train":1,
"Kc":20,
"T":"fft",
"anntype":"coordinate",
"batch_size":1000,
"dataset":"SDD",
"depth":16,
"deterministic":0,
"epochs":200,
"feature_dim":128,
"interval":0.4,
"key_points":"4_8_11",
"log_dir":"./logs/20240111-151105adaptive_lr2e-4_evspcsdd",
"loss":"keyl2",
"lr":0.0002,
"max_agents":50,
"model":"evspc",
"model_name":"adaptive_lr2e-4_",
"model_type":"agent-based",
"obs_frames":8,
"only_process_trajectory":0,
"pmove":-1,
"pred_frames":12,
"preprocess":"100",
"save_base_dir":"./logs",
"split":"sdd",
"step":1.0,
"#---------- SocialCircleArgs ----------":1,
"Ts":"none",
"partitions":8,
"rel_speed":0,
"use_direction":1,
"use_distance":1,
"use_move_direction":0,
"use_velocity":1,
"#---------- PhysicalCircleArgs ----------":2,
"adaptive_fusion":1,
"vision_radius":2.0}
``` 

I hope these could help!
C W

**Comment by LinneLee** [comment]
Thank you for the reference and suggestions! I'll give a try :)
